.github/workflows/macM1-e2e.yaml

```diff
@@ -0,0 +1,50 @@
+name: e2e-MacM1
+
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: '0 9 * * 1-5'
+
+
+jobs:
+
+  e2e-tests:
+    timeout-minutes: 45
+    runs-on: ProvoMacM1
+    env:
+      M1: 1
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+          ref: main
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '^1.18'
+      - name: Install dependencies
+        run: npm ci
+      - name: Post install script
+        run: npm run postinstall
+      - name: add tools to PATH
+        run: |
+          mkdir -p ~/.rd/bin
+          cp -R ~/actions-runner/_work/rancher-desktop/rancher-desktop/resources/darwin/bin ~/.rd/bin
+          export PATH="~/.rd/bin:$PATH"
```

Comment:
This should not be necessary — I thought you filed an issue to resolve this later, but I don't see it.

Reply:
I have filled the issue is in #2915.